pilot/prompts/development/parse_task.prompt

```diff
@@ -22,15 +24,14 @@ Examples:
     {
       "type": "save_file",
       "save_file": {
-        "name": "server.js",
-        "path": "/server.js",
+        "path": "./server.js",
         "code_change_description": "",
       },
     },
     {
       "type": "command",
       "command": {
-        "command": "ls",
+        "command": "mv ./index.js ./public/index.js"",
```

Comment:
this is inconsistent with the expected success message and command id, could confuse the llm